packages/system/vertical-pod-autoscaler-crds/Makefile

```
@@ -3,5 +3,17 @@ export NAMESPACE=cozy-$(NAME)
 include ../../../hack/package.mk
 
+test:
+	helm unittest .
+
+# Stamp helm.sh/resource-policy: keep on every CustomResourceDefinition so a
+# chart uninstall never garbage-collects the CRDs and, with them, every
+# VerticalPodAutoscaler and VerticalPodAutoscalerCheckpoint in the cluster.
+# The stamp goes into the CRD's existing metadata.annotations block; a CRD that
+# arrives without one is skipped, and the chart's helm-unittest suite is what
+# catches that.
 update:
```

checkmake (0.3.2)

[warning] Required target "all" is missing from the Makefile.

(minphony)